analysis/webservice/algorithms/doms/BaseDomsHandler.py

```diff
@@ -94,6 +94,10 @@ def __init__(self, results=None, args=None, bounds=None, count=None, details=Non
         self.__details = details
         self.__executionId = str(executionId)
 
+        # Add page num and size to details block
+        self.__details['pageNum'] = page_num
+        self.__details['pageSize'] = page_size
```

Comment:
Need to make sure this is not `None`

Reply:
Thanks for your review @RKuttruff. I don't think this will ever be `None`, as we provide defaults if the user doesn't provide these values

Reply:
In the issue that #263 fixed it was `None` because of the type of results that class returned. It may be worthwhile to just precede that with a quick check like
```python
if self.__details is None:
    self.__details = {}
```

Reply:
Ah I see, you meant check if `__details` was None, I had assumed you meant to check if page_num/page_size is None. Agreed, that seems wise